django/home/models.py

```diff
@@ -940,6 +940,32 @@ def __str__(self):
         return "submission {} by {}".format(self.title, self.submitter)
 
 
+class DigestArchive(index.Indexed, models.Model):
```

Comment:
Might be good to also include a OneToMany of Contributors to keep track of the people who worked on the Digest.

Reply:
also, a doi or doi_url might be a good future-proof field in the event that we ever start DOI-ing each comses digest

Reply:
as well as title / summary fields so we can have a slightly better display in the comses.net/digest list view

Reply:
Sorry for the spam! I should stop late night reviewing. Let's nix the title / summary idea, seems unnecessary on second thought

Reply:
As for Contributors: mostly (all?) John Murphy for the early Digests. In the more recent ones we've had a rotating guest editorship across the comses executive board but tends to default to Sean Bergin if we can't find anyone. Manuela and Kelly also do a lot of work to put it together, we should definitely include them as well as Laura Swantek for some of the previous digests. We don't have to go all out on this and can fill in the gaps later too.

Was trying to think up metadata that could spruce up the list of Digests (publish date, list of contributors, etc)

Reply:
Sounds good, contributors is just a blank m2m now. Editor seems to be named on each one so I suppose we could start with that, just need to think of the best way to input this now and in the future, I suppose an index file would work